pkg/policy/json.go

```diff
@@ -0,0 +1,221 @@
+package policy
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/netmap"
+)
+
+type (
+	filter struct {
+		Name    string   `json:"name,omitempty"`
+		Key     string   `json:"key,omitempty"`
+		Op      string   `json:"op,omitempty"`
+		Value   string   `json:"value,omitempty"`
+		Filters []filter `json:"filters,omitempty"`
+	}
+	replica struct {
+		Count    uint32 `json:"count"`
+		Selector string `json:"selector,omitempty"`
+	}
+	selector struct {
+		Count     uint32 `json:"count"`
+		Attribute string `json:"attribute"`
+		Filter    string `json:"filter,omitempty"`
+		Name      string `json:"name,omitempty"`
+		Clause    string `json:"clause,omitempty"`
+	}
+	placement struct {
+		Replicas  []replica  `json:"replicas"`
+		CBF       uint32     `json:"backup_factor,omitempty"`
```

Comment:
Let's keep it `container_backup_factor` everywhere. It may be confused with object backup factor or some other meaning.